CONTRIBUTING.md

```diff
@@ -417,17 +419,17 @@ wherever code is called that uses that citation (for example, in functions or in
 
 ### Installation
 
-Installation of PyBaMM and its dependencies is handled via [pip](https://pip.pypa.io/en/stable/) and [setuptools](http://setuptools.readthedocs.io/). It uses `CMake` to compile C++ extensions using [`pybind11`](https://pybind11.readthedocs.io/en/stable/) and [`casadi`](https://web.casadi.org/). The installation process is described in detail in the [source installation](https://docs.pybamm.org/en/latest/source/user_guide/installation/install-from-source.html) page and is configured through the `CMakeLists.txt` file.
+Installation of PyBaMM and its dependencies is handled via [pip](https://pip.pypa.io/en/stable/) and [scikit-build-core](https://scikit-build-core.readthedocs.io/en/latest/getting_started.html). It uses `CMake` to compile C++ extensions using [`pybind11`](https://pybind11.readthedocs.io/en/stable/) and [`casadi`](https://web.casadi.org/). The installation process is described in detail in the [source installation](https://docs.pybamm.org/en/latest/source/user_guide/installation/install-from-source.html) page and is configured through the `CMakeLists.txt` file.
 
 Configuration files:
 
 ```
-setup.py
 pyproject.toml
-MANIFEST.in
+.gitignore
 ```
 
-Note: `MANIFEST.in` is used to include and exclude non-Python files and auxiliary package data for PyBaMM when distributing it. If a file is not included in `MANIFEST.in`, it will not be included in the source distribution (SDist) and subsequently not be included in the binary distribution (wheel).
+Note: `pyproject.toml` and `.gitignore` are used to include and exclude non-Python files and auxiliary package data for PyBaMM when distributing it. If a file is mentioned in `.gitignore`, it will not be included in the source distribution (SDist) by `scikit-build-core`, and subsequently not be included in the binary distribution (wheel).
```

Comment:
```suggestion
Note: `pyproject.toml` and `.gitignore` are used to include and exclude non-Python files and auxiliary package data for PyBaMM when distributing it. If a file is mentioned in `.gitignore`, it will not be included in the source distribution (SDist), and subsequently not be included in the binary distribution (wheel).
```

The build frontend (`pip wheel` or `build`) is what invokes the build backend (`scikit-build-core`), but I think it's better to exclude that detail, at least from our docs.